book/controller.rst

```diff
@@ -429,9 +429,7 @@ A great way to see the core functionality in action is to look in the
 Redirecting
 ~~~~~~~~~~~
 
-If you want to redirect the user to another page, use the
-:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::redirect`
```

Comment:
this should be changed IMO. Protected methods are extension point, so they should be documented in the API doc too (private methods should indeed not appear)